RNAudioRecorderPlayer.podspec

```diff
@@ -11,6 +11,7 @@ Pod::Spec.new do |s|
   s.authors      = package['author']
   s.homepage     = package['homepage']
   s.platform     = :ios, "11.0"
+  s.ios.deployment_target = "9.0"
```

Comment:
Shouldn't the deployment target be 10.0 now?

Reply:
Yes should bo `10.0` now according to #334. I'll change this soon if there isn't a `PR` given for few days.